pkg/informer/informer.go

```diff
@@ -30,6 +43,50 @@ type GenericInformer struct {
 	retries       int64             // Counts times we have tried without establishing a watch.
 }
 
+///////////////////////////////////////////////////////////////////
+
+//leaving this here incase we want it to use other fields in future:
+
+// type Config struct {
+// 	APIVersion string `yaml:"apiVersion"`
+// 	Kind       string `yaml:"kind"`
+// 	Metadata   struct {
+// 		Name      string `yaml:"name"`
+// 		Namespace string `yaml:"namespace"`
+// 	} `yaml:"metadata"`
+// 	Data struct {
+// 		AllowedResources string `yaml:"AllowedResources"`
+// 		DeniedResources  string `yaml:"DeniedResources"`
+// 	} `yaml:"data"`
+// }
+
+type AllowedResources struct {
+	ApiGroups []string `yaml:"apiGroups"`
+	Resources []string `yaml:"resources"`
+}
+
+type DeniedResources struct {
+	ApiGroups []string `yaml:"apiGroups"`
+	Resources []string `yaml:"resources"`
+}
```

Comment:
These are the same, so make it generic.  Then when you create an instance the variable name will indicate allow or deny.